code/modules/atmospherics/machinery/components/binary_devices/pressure_valve.dm

```diff
@@ -31,6 +31,7 @@
 	if(can_interact(user))
 		target_pressure = MAX_OUTPUT_PRESSURE
 		investigate_log("was set to [target_pressure] kPa by [key_name(user)]", INVESTIGATE_ATMOS)
+		to_chat(user, "<span class='notice'>You maximize the pressure output on [src].</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='notice'>You set trigger input pressure on [src] to [target_pressure] kPa.</span>")
```

Reply:
> this not max possible pressure settings

What do you mean by this?

Reply:
from menu you can set max pressure to 100 atmospheres, but this make it mostly closed, until you make fusion in pipenet or somethis similar, so on alt click it set to near 12 atmosferes

Reply:
So you're saying that MAX_OUTPUT_PRESSURE is in fact not the max output pressure for this device? Seems like the better fix then is to set the actual max here rather than correct the message.

Reply:
It sets the pressure at which gas will flow through. he's saying that the wording doesn't match that behavior.